src/ui-test/camelk_java_debug_test.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+import path = require('path');
+import * as pjson from '../../package.json';
+import {
+	ActivityBar,
+	CustomTreeSection,
+	EditorView,
+	InputBox,
+	SideBarView,
+	TextEditor,
+	ViewItem,
+	VSBrowser,
+	Workbench
+} from 'vscode-extension-tester';
+import { Marketplace } from 'vscode-uitests-tooling';
+import { prepareEmptyTestFolder } from './utils/resourcesUtils';
+import { extensionIsActivated, sectionHasItem } from './utils/waitConditions';
+import * as uiTestConstants from './utils/uiTestConstants';
+import { promiseFail, promiseSucceed } from './utils/promiseUtils';
+
+const TEST_FOLDER = '../../../testFolder';
+const WORKSPACE_FOLDER = path.join(__dirname, TEST_FOLDER);
+
+const START_DEBUG_LABEL = uiTestConstants.startDebug;
+const REMOVE_INTEGRATION_LABEL = uiTestConstants.integrationRemove;
+
+describe.only('Test Debug on Camel K Integrations from Side Bar', function () {
+
+	before(async function () {
+		this.timeout(90000);
+		this.retries(3);
+		await VSBrowser.instance.waitForWorkbench;
```

Comment:
Sonar is reporting: `Refactor this redundant 'await' on a non-promise.sonarlint(typescript:S4123)`

i think it shoudl be written `await VSBrowser.instance.waitForWorkbench()`